portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,187 @@
-#content {
+.center {
+  display: block;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 50%;
```

Comment:
This will work for this purpose, but you may run into some issues in future changes. This will give your image a width corresponding to 50% of it's parent element, which probably isn't what you'd want in general. I'd recommend use a fixed length unit (see https://www.w3schools.com/cssref/css_units.asp). Percentages are based on the size of the parent, which can frequently change as you add layers of containers. For images I usually go with fixed px (pixels) or em (width of an "m" in the current font). However, vw and vh are also very useful when you want to support different sized screen. They represent 1% of the windows width/height, so "width: 50vw" should give you the same behavior you have now

Reply:
Okay, thank you for the information!